.github/workflows/artifacts-workflow.yml

```diff
@@ -0,0 +1,41 @@
+on:
+  push:
+    branches:
+    - master
+  pull_request:
+
+name: 'Artifacts'
+jobs:
+  chore:
+    name: 'Generating Artifacts'
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@master
+    - run: |
+        git fetch --no-tags --unshallow origin HEAD master
+
+    - name: 'Use Node.js 14.x'
+      uses: actions/setup-node@master
+      with:
+        node-version: 14.x
+
+    - name: 'Build the standard bundle'
+      run: |
+        node ./scripts/run-yarn.js build:cli
```

Comment:
I wonder if we could use that to speed up `yarn set version from sources` 🤔

Reply:
It looks like there's no public access to the GitHub Actions artifacts, so I don't think so. (https://github.community/t/public-read-access-to-actions-artifacts/17363/12)

If we could somehow download the artifacts without auth, maybe we could add some kind of "if it's our repo, check if there are artifacts for a commit and use them instead of building the repo" rule to `yarn set version from sources`, but that probably requires accessing quite a few endpoints to get all the needed data (I'm not sure if it's even possible). https://developer.github.com/v3/actions/artifacts/#list-artifacts-for-a-repository